### PR TITLE
Added deployment instructions to README.md (CU-d8rfw8)

### PR DESCRIPTION
**Pull request by schwarrrtz**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-d8rfw8 Add deployment instructions to README](https://app.clickup.com/t/d8rfw8)

File: README.md [hunk 1]
```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+# How to release a new version & update the entire Button system (server and RPi fleet)
+
+1. decide on an appropriate version number for the new version
+
+1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+
+1. make a new git tag in both the `BraveButtons` and `BraveButtonsConfig` repositories with the version number
```
Comment (trisapeace) [inline README.md:116]: Could you please include an example git command for this just to make sure that we're always tagging in the same way?

File: README.md [hunk 2]
```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
...
+
+1. update the `production` branch, ensuring that it is in sync with the tag for the new version
```
Comment (trisapeace) [inline README.md:118]: Could you please include the git command for this too to make sure that we're always doing it the same way and don't end up with any icky merge commits in weird places?

File: README.md [hunk 3]
```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+# How to release a new version & update the entire Button system (server and RPi fleet)
```
Comment (trisapeace) [inline README.md:110]: Can we specify that this is for production?

File: README.md [hunk 4]
```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
...
+1. cd into the `BraveButtons/server` directory
+
+1. shut down the server process: `sudo pm2 stop BraveServer`
+
+1. update the RPi fleet as described above (in a separate ssh session)
+
+1. pull the latest production code: `git checkout production && git pull origin production`
+
+1. run the server setup script: `sudo ./setup_server.sh ./.env`
+
```
Comment (trisapeace) [inline README.md:131]: Is it worth adding a verification step at the end to check on both dashboards to see that everything looks as expected?

**Review by trisapeace — APPROVED** [review]
Looks great. We can try following them for the next release.

I included some suggestions, but I approve even if you don't make any changes.

**Comment by schwarrrtz** [comment]
thanks for the quick review @trisapeace - I'll update this with your suggestions.

**Comment by trisapeace** [comment]
Oh! Don't forget the CHANGELOG :smile:

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,7 @@ the code was deployed.
 
 ## [Unreleased]
 ### Added
+- Deployment instructions in the README file.
```
Comment (trisapeace) [inline CHANGELOG.md:13]: Can you please add the ClickUp number here?

**Review by trisapeace — APPROVED** [review]
Looks awesome.

File: CHANGELOG.md (hunk 5 above)
Comment (schwarrrtz) [reply CHANGELOG.md:13]: Oops, somehow I was convinced there wasn't a ticket.

**Comment by mariocimet** [comment]
These look great!

**Review by mariocimet — APPROVED** [review]

**Pull request merged**